06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,183 @@
+const validFile = require('./books/validFile.json');
```

Comment:
I wasn't talking about the JSON file... MyBad!

I meant the name of your test file.